Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -1555,13 +1555,11 @@ def MultilinearMap.curryRight (f : MultilinearMap R M M₂) :
   map_update_add' := @fun dec m i x y => by
     rw [Subsingleton.elim dec (by clear dec; infer_instance)]; clear dec
     ext z
-    change f (snoc (update m i (x + y)) z) = f (snoc (update m i x) z) + f (snoc (update m i y) z)
-    rw [snoc_update, snoc_update, snoc_update, f.map_update_add]
+    simp
```

Comment:
I think here is another place where the `change` + `rw` makes it a little clearer what's going on.